jaeger/charts/linkerd-jaeger/templates/namespace-metadata-rbac.yaml

```diff
@@ -76,3 +83,4 @@ subjects:
 - kind: ServiceAccount
   name: namespace-metadata
   namespace: {{.Release.Namespace}}
+{{-end}}
```

Comment:
Please make sure your changes don't remove the required newline character at the end of the file.